x-pack/packages/lens/icons/src/assets/index.scss

```diff
@@ -0,0 +1,15 @@
+.lensChartIcon__subdued {
+  fill: $euiTextSubduedColor;
+
+  .lnsLayerChartSwitch__item-isSelected & {
+    fill: currentColor;
+  }
+}
+
+.lensChartIcon__accent {
+  fill: $euiColorVis0;
+
+  .lnsLayerChartSwitch__item-isSelected & {
+    fill: makeGraphicContrastColor($euiColorVis0, $euiColorDarkShade);
+  }
```

Comment:
Same question here. If this class isn't being used, we can likely remove these styles.